addons/hearing/functions/fnc_firedNear.sqf

```diff
@@ -53,9 +53,10 @@ TRACE_6("mag",_magazine,_weapon,_muzzle,_ammo,_firer,_gunner);
 
 private _vehAttenuation = [GVAR(playerVehAttenuation), 1] select (isNull objectParent ACE_player || {isTurnedOut ACE_player});
 private _loudness = _magazine call FUNC(getAmmoLoudness);
+private _hearingDamageFactor = [configFile >> "CfgAmmo" >> _ammo >> QGVAR(hearingDamageFactor), "NUMBER", 1] call CBA_fnc_getConfigEntry;
```

Comment:
Maybe put into `getAmmoLoudness` so it's cached and not looked up for every shot

Reply:
Initially didn't do it because I saw the equation below, but having actually taken a look at it I imagine it should work out to be fine.